netebpfext/net_ebpf_ext.c

```diff
@@ -404,6 +410,49 @@ net_ebpf_ext_unregister_callouts(void)
             FwpsCalloutUnregisterById(_net_ebpf_ext_wfp_callout_state[index].assigned_callout_id);
         }
     }
+
+    FwpsInjectionHandleDestroy(_net_ebpf_ext_l2_injection_handle);
+}
+
+static void
+_net_ebpf_ext_l2_inject_complete(
+    _In_ const void* context, _Inout_ NET_BUFFER_LIST* packet_clone, _In_ BOOLEAN dispatch_level)
+{
+    UNREFERENCED_PARAMETER(context);
+    FwpsFreeCloneNetBufferList(packet_clone, dispatch_level);
+}
+
+#define L2_INTERFACE_INDEX FWPS_FIELD_INBOUND_MAC_FRAME_NATIVE_INTERFACE_INDEX
+#define L2_NDIS_PORT FWPS_FIELD_INBOUND_MAC_FRAME_NATIVE_NDIS_PORT
```

Comment:
Why are these defines needed instead of just using the values on the right?  Do expect them to be defined to other things in the future?  (If so, add a // comment)

Reply:
Made the statements unnecessarily long. Since thats not a technical argument, I am reverting it.